server/api/categoryRouter.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Router for item actions
+ */
+const Category = require('../db/models/category');
+const express = require('express');
+const Item = require('../db/models/item');
+const router = express();
+router.use(express.json());
+
+/**
+ * Create a new category
+ * 
+ * req.body: {title: String}
+ * 
+ * response: {success: boolean, message: String}
+ */
+router.post('/createCategory', (req, res, next) => {
```

Comment:
next must be passed to these functions, as it refers to the callback function. There is a reference error if it's not included

Reply:
oh okay! will add it to item router too then

Reply:
Just added all of them:)))